Jenkinsfile

```diff
@@ -0,0 +1,178 @@
+pipeline {
+    agent any
+
+    environment {
+        APP_NAME = "square-one"
+        GIT_REPO = "moderntribe/${APP_NAME}.git"
+        BUILD_FOLDER = "./build"
+        DEPLOY_FOLDER = "./deploy"
+        GITHUB_TOKEN = credentials('tr1b0t-github-api-token')
+        JENKINS_VAULTPASS = "${env.APP_NAME}-vaultpass"
+        HOST_SSH_KEYS = "${env.APP_NAME}-ssh-key"
+        SLACK_CHANNEL = 'squareone'
+        ENVIRONMENT = environment()
+    }
+
+    stages {
+        // SCM
+        stage('Build SCM'){
+           steps {
+                echo "${env.BRANCH_NAME} - ${env.SLACK_CHANNEL} - ${env.ENVIRONMENT}"
+                slackSend(channel: "${SLACK_CHANNEL}", message: "Pipeline: (<${RUN_DISPLAY_URL}|#${BUILD_NUMBER}>)\nDeployment of `${APP_NAME}` branch `${env.BRANCH_NAME}` to `${env.ENVIRONMENT}` STARTED.")
+               // checkout scm
+                checkout([$class: 'GitSCM',
+                    branches: [[name: "${env.BRANCH_NAME}" ]],
+                    extensions: [[$class: 'WipeWorkspace'], [$class: 'RelativeTargetDirectory',  relativeTargetDir: BUILD_FOLDER]],
+                    userRemoteConfigs: [[url: "git@github.com:${env.GIT_REPO}", credentialsId: "jenkins-ssh-key"]]
+                ])
+            }
+        }
+        stage('Deploy SCM') {
+            steps {
+                // Decrypt values
+                withCredentials([string(credentialsId: "${JENKINS_VAULTPASS}", variable: 'vaultPass')]) {
+                    sh script: "echo '${vaultPass}' > ./.vaultpass", label: "Write vaultpass to local folder"
+                    sh script: "ansible-vault decrypt ${env.BUILD_FOLDER}/dev/deploy/.host/config/${env.ENVIRONMENT}.cfg.vaulted --output=${env.BUILD_FOLDER}/dev/deploy/.host/config/${env.ENVIRONMENT}.cfg --vault-password-file ./.vaultpass", label: "Decrypt config config file"
+                    sh 'rm ./.vaultpass'
+                }
+
+                // Load Host environment variables
+                loadEnvironmentVariables("${env.BUILD_FOLDER}/dev/deploy/.host/config/${env.ENVIRONMENT}.cfg")
+
+                // checkout scm WPEngine
```

Comment:
Might want to change this to `WPEngine, Pantheon, or similar git-based deployment`